src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -108,7 +108,16 @@
 	private final JLabel but5 = new JLabel(TiledImageCache.workspaceCode);
 	private final JLabel but5a = new JLabel(TiledImageCache.workspaceToggle);
 	private final JLabel but6 = new JLabel(TiledImageCache.workspaceModElementIDs);
-
+	
+	private final JMenuItem openElement = new JMenuItem(L10N.t("workspace.elements.list.edit.open"));

```

Comment:
Not really important, but you should create a new method inside the L10N class, to create a new JPopMenu, the same way it's already made with buttons, checkboxes and labels.

Reply:
i didn't really check about these stuffs, i only know L10N class is used for localizations stuff, i can see later